recipe/meta.yaml

```diff
@@ -7,25 +7,27 @@ package:
 source:
   # The source url changed for  0.37.*
   url: https://pypi.io/packages/source/S/SQLAlchemy-Utils/SQLAlchemy-Utils-{{ version }}.tar.gz
-  sha256: a6aaee154f798be4e479af0ceffaa5034d35fcf6f40707c0947d21bde64e05e5
+  sha256: a2181bff01eeb84479e38571d2c0718eb52042f9afd8c194d0d02877e84b7d74
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv
+  # Intervals is not available on s390x
+  skip: true  # [py<36 or s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed -vvv
 
 requirements:
   host:
     - python
     - pip
     - wheel
     - setuptools
+    - sqlalchemy 1.4.39
+    - importlib_metadata 6.0.0 # [py<38]
```

Comment:
§I'm curious, is there a particular reason to hard pin 6.0.0?

Reply:
For context, from what I see in the code, any version would work.

Reply:
The host section requires exact pinnings, and I defaulted to what we have on aggregate.

Reply:
I'm going off what the linter requires

Reply:
Actually, I think `importlib_metadata` should be a runtime dependency.

Reply:
Yeah, it's a runtime dependency without pinning, see https://github.com/kvesteri/sqlalchemy-utils/blob/fd94d17621b8e741b8c7353b700c19a4f38c7015/setup.py#L79
In host, python packages can be without a pinning, see https://github.com/anaconda-distribution/perseverance-skills/blob/a1d9216c91911a7a039765a57d5acc5bd2d2847f/sections/02_Package_building/02_Recipes/Recipe_standards.md?plain=1#L82

Reply:
Let me know if I've fixed things correctly.